client/ayon_core/hosts/aftereffects/api/__init__.py

```diff
@@ -9,7 +9,6 @@
 )
 
 from .pipeline import (
-    AfterEffectsHost,
```

Comment:
I wonder if instead of removing this it should actually be added to the `__all__` below instead? (Looking at what the `api/__init__.py` seems to do for e.g. Blender or Maya hosts.